app/src/main/java/com/example/android/tvleanback/ui/PlaybackOverlayFragment.java

```diff
@@ -154,18 +159,25 @@ public void onItemSelected(Presenter.ViewHolder itemViewHolder, Object item,
         setOnItemViewClickedListener(new ItemViewClickedListener());
     }
 
+
     @Override
-    public void onAttach(Activity activity) {
-        super.onAttach(activity);
-
-        // This makes sure that the container activity has implemented
-        // the callback interface. If not, it throws an exception
-        try {
-            mCallback = (OnPlayPauseClickedListener) activity;
-        } catch (ClassCastException e) {
-            throw new ClassCastException(activity.toString()
-                    + " must implement OnPlayPauseClickedListener");
+    public void onStart() {
+        super.onStart();
+
+        mMediaController = getActivity().getMediaController();
```

Comment:
Add a comment here about how the Fragment can get the MediaController from the Activity. This is an important step that isn't obvious (it's a Lollipop API).